xilem_core/src/views/memoize.rs

```diff
@@ -115,16 +126,101 @@ where
     }
 }
 
-/// Memoize the view, until the `data` changes (in which case `view` is called again)
-pub fn memoize<State, Action, Context, Message, Data, V, ViewFn>(
-    data: Data,
-    view: ViewFn,
-) -> Memoize<Data, ViewFn>
+/// Specialized version of [`Memoize`], which doesn't take any data at all, the closure is evaluated only once and when a child view forces a rebuild
+pub struct Frozen<InitView, State, Action> {
+    init_view: InitView,
+    phantom: PhantomData<fn() -> (State, Action)>,
+}
+
+/// Specialized version of [`memoize`], which doesn't take any data at all, the closure is evaluated only once and when a child view forces a rebuild
```

Comment:
We should explain this in terms of why you would use it, not in reference to any other value.

Reply:
Probably skim again, I also added an example doc-comment for `Memoize`